tiling.js

```diff
@@ -2839,13 +2845,31 @@ function grabEnd(metaWindow, type) {
     inGrab = false;
 }
 
+function setAllWorkspacesInactive() {
+    try {
```

Comment:
my thoughts on the try catch: not really needed (although isn't hurting) - you already do a check on
```javascript
if (wm) {...
```